store/pogreb/pogreb.go

```diff
@@ -196,3 +215,14 @@ func (s *pStorage) removeEntry(k []byte, value entry.Value, stored []entry.Value
 	}
 	return false, nil
 }
+
+func (s *pStorage) lock(k []byte) error {
+	if !s.mlk.TryLock(string(k)) {
+		return errors.New("couldn't get the lock after maxRetries")
```

Comment:
This is just not an actionable error.  If it ever happened it would cause an occasional failure that would be difficult to detect and impossible to fix, other than to keep retrying until it finally works... in which case why ever time out.